types/node-siege/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for node-siege 0.0
+// Project: https://github.com/npr/node-siege
+// Definitions by: Zlatko Andonovski <https://github.com/Goldsmith42>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { EventEmitter } from 'events';
+
+declare class Siege extends EventEmitter {
+    args(args: string): Siege;
+    run(): Siege;
```

Comment:
Run requires args, you can use TS var args here:
https://github.com/npr/node-siege/blob/master/lib/siege.js#L26
either `any[]` or generic ones:
https://www.typescriptlang.org/docs/handbook/generics.html#hello-world-of-generics